src/Reflection/ReflectionClass.php

```diff
@@ -122,47 +116,34 @@ public function __get($key)
      * @param mixed $value
      * @return void
      */
-    public function __set($key, $value)
+    public function __set(string $key, $value): void
     {
         $this->config[$key] = $value;
     }
 
-    /**
-     * Return array of dispatchable {@link \Laminas\Server\Reflection\ReflectionMethod}s.
-     *
-     * @access public
-     * @return array
-     */
-    public function getMethods()
+    public function getMethods(): array
     {
         return $this->methods;
     }
 
-    /**
-     * Get namespace for this class
-     *
-     * @return string
-     */
-    public function getNamespace()
+    public function getNamespace(): ?string
     {
         return $this->namespace;
     }
 
     /**
      * Set namespace for this class
      *
-     * @param string $namespace
      * @throws Exception\InvalidArgumentException
-     * @return void
      */
-    public function setNamespace($namespace)
+    public function setNamespace(?string $namespace = null): void
```

Comment:
the same as before, we should not have default value here, we should decide if we want to support null, or just empty string for unsetting the namespace

Reply:
Agreed. Default value removed.